apps/journeys-admin/src/components/Editor/VideoLibrary/VideoDetails/VideoDetails.tsx

```diff
@@ -67,19 +87,56 @@ export function VideoDetails({
   const [loadVideo, { data, loading }] = useLazyQuery<GetVideo>(GET_VIDEO, {
     variables: { id }
   })
+  const [updateVideoBlockNextStep] = useMutation<UpdateVideoBlockNextStep>(
+    UPDATE_VIDEO_BLOCK_NEXT_STEP
+  )
+  const { journey } = useJourney()
+  const {
+    state: { selectedStep, selectedBlock }
+  } = useEditor()
 
   const handleChange = (selectedIds: string[]): void => {
     setSelectedIds(selectedIds)
   }
 
-  const handleVideoSelect = (): void => {
+  const handleNextStepBlock = async (): Promise<void> => {
```

Comment:
```suggestion
  const updateDefaultNextStep = async (): Promise<void> => {
```
Or something that spells out that it's we're only dealing with "default" nextBlockId and if we use handle - make it the format `handle[name][Action]`